docs/sources/next/javascript-api/k6-experimental/redis/redis-options.md

```diff
@@ -1,55 +1,66 @@
 ---
-title: 'Redis options'
+title: 'Options'
```

Comment:
I think we generally want the title property and the h1 heading for the page to match. Is there any reason why you want to update this to just "Options"? 🤔